frontend/src/components/UserDetail/EmployeeDetail.tsx

```diff
@@ -296,7 +296,11 @@ const EmployeeDetail = () => {
             netId={employee.netId}
             employee={employee}
             role={roleValue()}
-            photoLink={employee.photoLink}
+            photoLink={
+              employee.photoLink
+                ? employee.photoLink + '?v=' + new Date().getTime()
```

Comment:
same comment as above - also this totally could go into a helper function because it's very specific behavior that if updated needs to be updated everywhere